.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: cpp
+language: c
 compiler:
   - gcc
   - clang
 
 # This is a lie: we don't need sudo but this is required to get an
 # Ubuntu image with a libc that isn't ancient, and with cmocka libs.
```

Comment:
We still need to install cmake to be able to build cmocka, but it looks like it works even with "sudo = false"